src/utils/hooks/useGetReviewInfo.tsx

```diff
@@ -152,4 +157,26 @@ const useGetReviewInfo = ({ applicationId, serial }: UseGetReviewInfoProps) => {
   }
 }
 
+const migrationCode_2_0_0 = (assignment: AssignmentDetails) => {
+  const {
+    current: { assignmentStatus },
+    assignedSections,
+    reviewQuestionAssignments,
+  } = assignment
+  if (assignmentStatus === ReviewAssignmentStatus.Assigned) {
+    // When Assigned without assignedSections and with assigned questions - showing incosistency
+    // due to previous versions (<2.0.0) wasn't using the assignedSections field. Update!
+    if (assignedSections.length === 0 && reviewQuestionAssignments.length !== 0) {
+      assignment.assignedSections = reviewQuestionAssignments.reduce(
+        (assignedSections: string[], { templateElement }) => {
+          const code = templateElement?.section?.code as string
+          if (!assignedSections.includes(code)) assignedSections.push(code)
+          return assignedSections.sort()
+        },
+        []
+      )
+    }
+  }
+}
+
```

Comment:
I don't think we need to start putting migration code in yet do we? It's still fairly trivial to just manually run an SQL command on existing databases.

Plus, this would be easier done in the back-end wouldn't it? Seems like it would be tidier to manually mutate the data ONCE rather than modify local state in front-end every time it's run.

I think we need to have a plan for adding migration code, I'm a bit reluctant to start adding more complexity to already complicated existing components.

For migration code, I'd probably want to have:
- as little as possible
- all in one script, preferably in the back-end
- mutates data once, so it doesn't have to make state conversions at runtime
- doesn't run as part of running app -- should be a start-up or manual thing only

Anyway, let's chat about it early next week?

Reply:
Totally agreed. So, I'll be removing this now and checking out [your PR](https://github.com/openmsupply/application-manager-server/pull/659) that moves it to back-end